.github/workflows/adelphi-python.yaml

```diff
@@ -0,0 +1,29 @@
+name: adelphi Python
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
Do we really need to specify the branch name?
We might change the name to trunk to match other repos and we'd have to change it here too.
Also, having it hardcoded won't it make it run less frequently, since all PRs are against master right now.

Reply:
Honestly I'm not sure if this field can be empty or not.  Most of the samples I found explicitly specified a target branch here but I didn't find any docs one way or the other.  That said, I guess I don't view this as a major burden.  If we decide to rename master at some point we know that'll be a disruptive change anyway; fixing this usage would just be one thing among many we'd have to address.  I also don't agree that all our PRs are off of master at the moment; it's certainly the case that most, even the overwhelming majority, are but there have been a few occasions where I've wanted review of a specific proposal for a feature branch already in-flight... and that leads to branches off of branches.  I readily agree it's not common but it does happen.

I'll do some checking to see whether a branch name is required here or not.